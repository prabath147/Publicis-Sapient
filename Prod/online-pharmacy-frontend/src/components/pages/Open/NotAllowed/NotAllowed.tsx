import { Container } from '@mantine/core'
import { Link } from 'react-router-dom'

export default function NotAllowed() {
    return (
        <Container>

            <br />
            <br />
            <h1>
                Not Allowed
            </h1>
            <br />
            <h3>
                <Link to="/">
                    Go to Home
                </Link>
            </h3>
        </Container>
    )
}
