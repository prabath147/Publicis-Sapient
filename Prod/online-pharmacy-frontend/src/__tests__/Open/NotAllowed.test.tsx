import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NotAllowed from '../../components/pages/Open/NotAllowed/NotAllowed';

it('component created', () => {
    const component = renderer.create(
        <BrowserRouter>
            <NotAllowed />
        </BrowserRouter>
    )
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})
