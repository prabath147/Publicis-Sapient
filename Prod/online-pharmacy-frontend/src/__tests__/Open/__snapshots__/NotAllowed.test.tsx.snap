// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component created 1`] = `
<div
  className="mantine-Container-root mantine-15pjuqq"
>
  <br />
  <br />
  <h1>
    Not Allowed
  </h1>
  <br />
  <h3>
    <a
      href="/"
      onClick={[Function]}
    >
      Go to Home
    </a>
  </h3>
</div>
`;
